sdk/python/kfp/_client.py

```diff
@@ -277,9 +289,32 @@ def set_user_namespace(self, namespace):
       namespace: kubernetes namespace the user has access to.
     """
     self._context_setting['namespace'] = namespace
+    if not os.path.exists(os.path.dirname(Client.LOCAL_KFP_CONTEXT)):
+        os.makedirs(os.path.dirname(Client.LOCAL_KFP_CONTEXT))
     with open(Client.LOCAL_KFP_CONTEXT, 'w') as f:
       json.dump(self._context_setting, f)
 
+  def get_kfp_healthz(self):
+    """Get healthz info of KFP deployment.
+
+    Returns:
+      response: json formatted response from the healtz endpoint.
+    """
+    healthz_api = 'http://' + self._existing_config.host + '/' + Client.HEALTH_PATH
+    count = 0
+    response = None
+    max_attempts = 60
```

Comment:
Do we really need to retry 60 times? If so, shall we rename the function to `wait_kfp_healthz`?

Reply:
This was mainly just to try and brute-force getting a response to see if retries would help. If this indeed seems necessary I think the retry should be handled when trying to set the namespace rather than here.